smartlearner/tasks/tests/test_stopping_criteria.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
```

Comment:
I find it cleaner than putting a test module everywhere. It's clutter autocomplete of imports.
But I admit that's a matter of taste.